src/amuse/community/mesa/interface.py

```diff
@@ -1,1545 +1,5 @@
-        if number_of_species is None:
-            number_of_species = self.get_number_of_species(indices_of_the_stars)
-        return self.get_mass_of_species(
-            [indices_of_the_stars]*number_of_species, 
-            list(range(1,number_of_species+1))
-        )
-    
-    def new_particle_from_model(self, internal_structure, current_age=0|units.Myr, key=None):
-        if isinstance(internal_structure, dict):
-            if "dmass" in internal_structure:
-                mass_profile = internal_structure['dmass'][::-1]
-            else:
-                cumulative_mass_profile = [0.0] | units.MSun
-                cumulative_mass_profile.extend(internal_structure['mass'])
-                mass_profile = (cumulative_mass_profile[1:] - cumulative_mass_profile[:-1])[::-1]
-            self.new_stellar_model(
-                mass_profile,
-                internal_structure['radius'][::-1],
-                internal_structure['rho'][::-1],
-                internal_structure['temperature'][::-1],
-                internal_structure['luminosity'][::-1],
-                internal_structure['X_H'][::-1],
-                internal_structure['X_He'][::-1],
-                internal_structure['X_C'][::-1],
-                internal_structure['X_N'][::-1],
-                internal_structure['X_O'][::-1],
-                internal_structure['X_Ne'][::-1],
-                internal_structure['X_Mg'][::-1],
-                internal_structure['X_Si'][::-1],
-                internal_structure['X_Fe'][::-1]
-            )
-        else:
-            if hasattr(internal_structure, "dmass"):
-                mass_profile = internal_structure.dmass[::-1]
-            else:
-                cumulative_mass_profile = [0.0] | units.MSun
-                cumulative_mass_profile.extend(internal_structure.mass)
-                mass_profile = (cumulative_mass_profile[1:] - cumulative_mass_profile[:-1])[::-1]
-            self.new_stellar_model(
-                mass_profile,
-                internal_structure.radius[::-1],
-                internal_structure.rho[::-1],
-                internal_structure.temperature[::-1],
-                internal_structure.luminosity[::-1],
-                internal_structure.X_H[::-1],
-                internal_structure.X_He[::-1],
-                internal_structure.X_C[::-1],
-                internal_structure.X_N[::-1],
-                internal_structure.X_O[::-1],
-                internal_structure.X_Ne[::-1],
-                internal_structure.X_Mg[::-1],
-                internal_structure.X_Si[::-1],
-                internal_structure.X_Fe[::-1]
-            )
-        tmp_star = datamodel.Particle(key=key)
-        tmp_star.age_tag = current_age
-        return self.imported_stars.add_particle(tmp_star)
-
-
-
-Mesa = MESA
+import amuse
+if amuse.config.community.mesa_version == "2208":
+    from amuse.community.mesa_2208.interface import *
+elif amuse.config.community.mesa_version == "new":
```

Comment:
"new" or definite version?

Reply:
new is a placeholder for a better name. 2208 is an exact version but I'm hoping "new" will also accept newer versions than the currently used one, so I don't want to use a number there... I don't like "new" because it won't always be new.